HackingWithSwift/First project/Project1/Project1/ViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ViewController.swift
+//  Project1
+//
+//  Created by Katasee on 26/09/2024.
+//
+
+import UIKit
+
+class ViewController: UITableViewController {
+    
+    var pictures = [String]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        title = "Storm Viewer"
+        navigationController?.navigationBar.prefersLargeTitles = true
+        
+        let fm = FileManager.default
+        let path = Bundle.main.resourcePath!
+        let items = try! fm.contentsOfDirectory(atPath: path)
+        
+    
+        for item in items {
+            if item.hasPrefix("nssl") {
+                pictures.append(item)
+            }
+        }
+        pictures.sort()
+        print(pictures)
```

Comment:
Don't leave print statements in production code. It should only be when you develop and need you need to check something.